ext/CUDAExt/implementations/peakflops_gpu.jl

```diff
@@ -70,10 +72,26 @@ function _theoretical_peakflops_gpu_tensorcores(;
     max_clock_rate = CUDA.attribute(device, CUDA.CU_DEVICE_ATTRIBUTE_CLOCK_RATE) # in kHz
     num_tensor_cores = ntensorcores(device)
     max_peakflops = max_clock_rate * num_tensor_cores * 1e-9 # in TFLOP/s
-    if devtype == :A100
+    if devtype == :Hopper
+        # matrix dimensions 8x8x4, factor 2 for nflops in A*B+C see
+        # * <https://resources.nvidia.com/en-us-tensor-core/gtc22-whitepaper-hopper> (figures 10-11)
+        # * <https://developer.nvidia.com/blog/nvidia-hopper-architecture-in-depth/> (figures 5-8)
         if Symbol(dtype) == :Float16
-            # matrix dimensions 8x8x4, factor 2 for nflops in A*B+C
-            # see e.g. https://peerj.com/articles/cs-330.pdf
+            max_peakflops *= 2 * 16 * 8 * 4 # XXX: Wrong result!
+        elseif Symbol(dtype) in (:Float32, :TensorFloat32, :TF32)
+            max_peakflops *= 2 * 8 * 8 * 4 # XXX: Wrong result!
+        elseif Symbol(dtype) == :Float64
+            max_peakflops *= 2 * 4 * 4 * 2
+        elseif Symbol(dtype) == :Int8
+            max_peakflops *= 2 * 2 * 32 * 8 * 4 # XXX: Wrong result!
```

Comment:
Maybe there's an extra factor of 2 in this formula, but I based this on the `Int8` calculation below